src/components/Editor/Footer.js

```diff
@@ -148,6 +151,18 @@ class SourceFooter extends PureComponent<Props> {
     );
   }
 
+  renderSourceSummary() {
+    const { selectedSource } = this.props;
+    const source = selectedSource.toJS();
+    console.log(source);
+    if (isOriginalId(source.id)) {
+      const bundleSource = getGeneratedSourceId(source.id);
+      console.log(bundleSource);
+      return ` (source mapped from ${getFilename(bundleSource)})`;
+    }
+    return "  yo";
+  }
```

Comment:
great start... lets see what works when we test w/ http://firefox-dev.tools/debugger-examples/examples/increment/